PyInstaller/lib/modulegraph/modulegraph.py

```diff
@@ -2719,8 +2719,14 @@ def get_operation_arg_name():
             list of names is the unsigned short (i.e., 2-byte integer) argument
             passed to the current bytecode operation.
             """
-            co_names_index = get_operation_arg()
-            return module_code_object.co_names[co_names_index]
+            try:
+                co_names_index = get_operation_arg()
+                return module_code_object.co_names[co_names_index]
+            except IndexError as e:
+                print('co_names_index: {}'.format(co_names_index))
```

Comment:
if `get_operation_arg()` fails, `co_names_index` is not set and this will raise another exception.